docs/source/install.md

```diff
@@ -133,4 +148,13 @@ Additionally, `graphviz` can be installed by running:
 brew install graphviz
 ```
 
+### Installing EvalML on an M1 Mac
+
+Not all of EvalML's dependencies support Apple's new M1 chip. For this reason, `pip` or `conda` installing EvalML will
+fail. The core set of EvalML dependencies can be installed in the M1 chip, so we recommend you install EvalML with core
+dependencies. 
+
+Alternatively, we have heard from users that you can install all of EvalML's dependencies on the M1 chip using a Rosetta
```

Comment:
nit: instead of "we have heard from users that you can" maybe something along the lines of "Alternatively, there is experimental support for M1 chips using a Rosetta terminal...". Sounds better IMO but up to your discretion!

Reply:
Good point - will make this change!